chasm/src/main/java/org/quiltmc/chasm/internal/asm/visitor/LocalInterpreter.java

```diff
@@ -85,7 +90,43 @@ protected boolean isAssignableFrom(Type type1, Type type2) {
                     return isAssignableFrom(type1.getElementType(), type2.getElementType());
                 }
                 if (type1.getSort() == Type.OBJECT && type2.getSort() == Type.OBJECT) {
-                    return context.isAssignable(type1.getInternalName(), type2.getInternalName());
+                    String leftClass = type1.getInternalName();
```

Comment:
Could all this code be extracted into a method maybe?

Reply:
Sure